clj/test/slipstream/ui/models/users_test.clj

```diff
@@ -83,3 +90,5 @@
 (expect
   parsed-metadata
   (-> raw-metadata-str u/clojurify-raw-metadata-str model/parse))
+
+(run-all-tests)
```

Comment:
Please do not commit the `(run-all-tests)` lines, because it makes the normal test execution behave unexpectedly. Can you please remove it?